src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/service/ClientsService.kt

```diff
@@ -118,6 +119,14 @@ class ClientsService(
       .build()
 
     registeredClientRepository.save(duplicatedRegisteredClient)
+    val authorizationConsent = authorizationConsentRepository.findById(AuthorizationConsentId(client.id, client.clientId))
```

Comment:
Could you use `findByIdOrNull` here? Would save messing around with the Optional